src/sentry/mediators/external_requests/alert_rule_action_requester.py

```diff
@@ -56,11 +57,13 @@ def _make_request(self) -> Mapping[str, Union[bool, str]]:
                     "error_message": str(e),
                 },
             )
-            message = f"{self.sentry_app.name}: {str(e.response.text) or DEFAULT_ERROR_MESSAGE}"
+            # NOTE: bool(e.response) is always False because non-2xx responses are Falsey.
+            text = e.response.text if e.response is not None else None
```

Comment:
Does this no longer need to be converted to a string?

Reply:
If we convert to string, then when `text` is `None` we'll get
```
Example Integration: None
```
instead of
```
Example Integration: Something went wrong!
```

Reply:
Might be slightly simpler to do `e.response.text if e.response is not None else DEFAULT_ERROR_MESSAGE`

Reply:
When `e.response.text` is `""` I think I still want to swap in `DEFAULT_ERROR_MESSAGE`.